config/routes.rb

```diff
@@ -21,6 +21,8 @@
 
     resources :event_levels, module: "event", controller: "levels", except: [:show]
 
+    resources :external_invoices, only: [:show], module: :people, param: :invoice_id
```

Comment:
I had to create it as a `resources` route here, since with just a `get ...` the log link would still point to the API endpoint.